rsmtool/test_utils.py

```diff
@@ -40,7 +40,9 @@ def check_run_experiment(source,
                          consistency=False,
                          skll=False,
                          file_format='csv',
-                         given_test_dir=None):
+                         given_test_dir=None,
+                         input_is_file=True,
```

Comment:
It seems like these two parameters (`input_is_file` and `config_obj_or_dict`) could be collapsed in the some way. Maybe getting rid of `input_is_file` and retaining only the `config_obj_or_dict` and then using its value:

```python
    if config_obj_or_dict is None:
        config_input = join(test_dir,
                           'data',
                           'experiments',
                           source,
                           '{}.json'.format(experiment_id))
    else:
        config_input = config_obj_or_dict
```